src/index.html

```diff
@@ -77,6 +77,9 @@ <h2>Calcite Component Examples - WIP !</h2>
         <li>
           <calcite-link href="/demos/fab">calcite-fab</calcite-link>
         </li>
+        <li>
+          <calcite-link href="/demos/calcite-graph.html">calcite-graph</calcite-link>
```

Comment:
Currently, `calcite-graph` [is bundled](https://github.com/Esri/calcite-components/blob/master/stencil.config.ts#L66) with the slider as a supporting component and not meant to be used standalone. 

If we need to make it public, we need to update the config bundles with an individual entry for the graph component. We should probably rename it as well since 'graph' is too general of a term for this. Maybe `calcite-histogram`? Not sure if this would lead us down a path of charting components. 

@subgan82 Would any of this conflict with your chart work?

Reply:
I reverted this change.